docs/install.md

```diff
@@ -936,3 +936,189 @@ This procedure can be used to downgrade the active version of SAT.
+    $ python3 -m venv ${SAT_VENV_PATH}
+    $ . ${SAT_VENV_PATH}/bin/activate
+    ```
+
+1.  Install the SAT Python package in the virtual environment.
+
+    ```screen
+    (venv) $ export PIP_EXTRA_INDEX_URL="https://artifactory.algol60.net/artifactory/csm-python-modules/simple"
+    (venv) $ git clone --branch=release/3.17 https://github.com/Cray-HPE/sat.git
+    (venv) $ pip install -r sat/requirements.lock.txt
+    ...
+    (venv) $ pip install ./sat
+    ...
+    ```
+
+1.  Optional: Add the `sat` virtualenv to the user's `PATH` environment variable.
+
+    If a shell other than `bash` is in use, replace `~/.bash_profile` with the appropriate profile path.
+
+    If the virtualenv is not added to the user's `PATH` environment variable, then
+    `source ${SAT_VENV_PATH}/bin/activate` will need to be run before running any SAT commands.
+
+    ```screen
+    (venv) $ deactivate
+    $ echo export PATH=\"${SAT_VENV_PATH}/bin:${PATH}\" >> ~/.bash_profile
+    $ source ~/.bash_profile
+    ```
+
+1.  Copy the file `/etc/kubernetes/admin.conf` from `ncn-m001` to `~/.kube/config` on the external system.
+
+    ```screen
+    $ mkdir -p ~/.kube
+    $ scp ncn-m001:/etc/kubernetes/admin.conf ~/.kube/config
+    admin.conf                                       100% 5566   3.0MB/s   00:00
+    ```
+
+1.  Add a new entry for the hostname `kubernetes` to the external system's `/etc/hosts` file.
+
+    The `kubernetes` hostname should correspond to the CAN IP address on `ncn-m001`. On CSM 1.2, this can be determined
+    by querying the IP address of the `bond0.cmn0` interface.
+
+    ```screen
+    $ ssh ncn-m001 ip addr show bond0.cmn0
+    13: bond0.cmn0@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether b8:59:9f:1d:d9:0e brd ff:ff:ff:ff:ff:ff
+    inet 10.102.1.11/24 brd 10.102.1.255 scope global vlan007
+       valid_lft forever preferred_lft forever
+    inet6 fe80::ba59:9fff:fe1d:d90e/64 scope link
+       valid_lft forever preferred_lft forever
+    $ IP_ADDRESS=10.102.1.11
+    ```
+
+    On CSM versions prior to 1.2, the CAN IP can be determined by querying the IP address of the `vlan007` interface.
+
+    ```screen
+    $ ssh ncn-m001 ip addr show vlan007
+    13: vlan007@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
+    link/ether b8:59:9f:1d:d9:0e brd ff:ff:ff:ff:ff:ff
+    inet 10.102.1.10/24 brd 10.102.1.255 scope global vlan007
+       valid_lft forever preferred_lft forever
+    inet6 fe80::ba59:9fff:fe1d:d90e/64 scope link
+       valid_lft forever preferred_lft forever
+    $ IP_ADDRESS=10.102.1.10
+    ```
+
+    Once the IP address is determined, add an entry to `/etc/hosts` mapping the IP address to the hostname `kubernetes`.
+
+    ```screen
+    $ echo "${IP_ADDRESS} kubernetes" | sudo tee -a /etc/hosts
+    10.102.1.11 kubernetes
+    ```
+
+1.  Modify `~/.kube/config` to set the cluster server address.
```

Comment:
Does it work to just put the IP address in without putting an entry in `/etc/hosts`? Like:

```
server: https://10.102.1.11:6443
```

I think that would be better because it saves a step and also doesn't require editing `/etc/hosts`.

Reply:
Adding the line to `/etc/hosts` is actually required to "fool" the certificate verification. The certificate isn't valid for the CAN IP address on the NCN:
```crayadm@kubetest-final:~> kubectl get nodes
Unable to connect to the server: x509: certificate is valid for 10.16.0.1, 10.252.1.12, 10.252.1.2, 10.252.1.11, not 10.102.1.13
```

Any of the Subject Alternative Names listed for the cert here would work:
```
crayadm@kubetest-final:~> openssl s_client -connect 10.102.1.13:6443 </dev/null 2>/dev/null | openssl x509 -noout -text | grep 'DNS'
                DNS:kubernetes, DNS:kubernetes.default, DNS:kubernetes.default.svc, DNS:kubernetes.default.svc.cluster.local, DNS:ncn-m001, IP Address:10.16.0.1, IP Address:10.252.1.12, IP Address:10.252.1.2, IP Address:10.252.1.11
```
I chose `kubernetes` to add to `/etc/hosts` since it seems most self-explanatory.

Probably a "better" solution would be to add a new SAN to the certificate for the system-specific hostname of `ncn-m001`. This would probably be a CASMPET issue, I can dig into this a bit more. I agree that my solution here is a bit janky, but it seems to work acceptably well.

Reply:
I'm good with it for now, just thinking that editing /etc/hosts is not always something people want to do on their local machines.

Reply:
Definitely agree on that, but there doesn't seem to be a better solution without having that SAN set up properly on the certificate. I wonder if there is a succinct way of noting this in the ticket. I suppose I could use the above command as a starting point to determine the hostname for the certificate, and then use that in place of `kubernetes` in the hosts file, or otherwise note that they should use `host <ip_address>` to check if there is a hostname associated with the NCN CAN IP address which is also a SAN on the certificate. It does get complicated quickly.